mediatek: correct address of MT753x switch IC

namiltd opened the pull request:
Continuation of commit 8b66f1a. Set the switch address on the MDIO bus to 31.
This is required for all boards currently working with the mt7530 DSA driver.

Fixes: #15419

neheb commented:
with this, target/linux/mediatek/dts/mt7623a-unielec-u7623-02.dtsi is left.

hauke commented on target/linux/mediatek/dts/mt7623a-unielec-u7623-02.dtsi:
```diff
@@ -121,26 +121,24 @@
 };
 
 &mt7530 {
-	compatible = "mediatek,mt7530";
```
Please keep the definition here, I think the switch configuration should be defined at one place. 
The change to 31 looks fine.

namiltd replied:
Done

namiltd committed:
mediatek: correct address of MT753x switch IC

Continuation of commit 8b66f1a. Set the switch address on the MDIO bus to 31.
This is required for all boards currently working with the mt7530 DSA driver.

Fixes: #15419
Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>
